specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/virtualNetworkTap.json

```diff
@@ -0,0 +1,356 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2018-08-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworkTaps/{tapName}": {
+        "delete": {
+            "tags": [
+                "VirtualNetworkTap"
+            ],
+            "operationId": "VirtualNetworkTaps_Delete",
+            "description": "Deletes the specified virtual network tap.",
+            "parameters": [
+                {
+                    "name": "resourceGroupName",
+                    "in": "path",
+                    "required": true,
+                    "type": "string",
+                    "description": "The name of the resource group."
+                },
+                {
+                    "name": "tapName",
+                    "in": "path",
+                    "required": true,
+                    "type": "string",
+                    "description": "The name of the virtual network tap."
+                },
+                {
+                    "$ref": "./network.json/parameters/ApiVersionParameter"
```

Comment:
Should be "$ref": "./network.json#/parameters/ApiVersionParameter"

Reply:
resolved